--- ChangeLog	2025-10-17 14:23:04.769894228 -0400
+++ ChangeLog-spell-corrected	2025-10-17 14:24:39.516750580 -0400
@@ -6,7 +6,7 @@

 	* admin-tools/make-dist-3.0-3.2.sh, admin-tools/make-dist-3.11.sh,
 	admin-tools/make-dist-3.3-3.5.sh,
-	admin-tools/make-dist-3.6-3.10.sh, admin-tools/make-dist-newest.sh:
+	admin-tools/make-dist-3.6-3.10.sh, admin-tools/make-dist-newest.sh:
 	Administrivia

 2025-10-17  rocky <rocky@gnu.org>
@@ -48,13 +48,13 @@

 2025-10-14  rocky <rocky@gnu.org>

-	* trepan/processor/command/list.py, trepan/processor/print.py:
+	* trepan/processor/command/list.py, trepan/processor/print.py:
 	Better handling of pyasm files pyasm lines have location information and newlines. So don't add
 	these.

 2025-10-11  rocky <rocky@gnu.org>

-	* pyproject.toml: Adjust develpment optional packages
+	* pyproject.toml: Adjust development optional packages

 2025-10-11  rocky <rocky@gnu.org>

@@ -85,12 +85,12 @@

 2025-09-30  rocky <rocky@gnu.org>

-	* admin-tools/make-dist-3.0-3.2.sh, admin-tools/make-dist-newest.sh:
+	* admin-tools/make-dist-3.0-3.2.sh, admin-tools/make-dist-newest.sh:
 	Packaging administrivia

 2025-09-30  rocky <rocky@gnu.org>

-	* pyproject.toml, trepan/lib/core.py, trepan/lib/stack.py:
+	* pyproject.toml, trepan/lib/core.py, trepan/lib/stack.py:
 	Administriva and add a guard check * Relax xdis version * Check for None in frame more often * Add more type annotations

 2025-09-27  rocky <rocky@gnu.org>
@@ -155,7 +155,7 @@
 2025-08-23  rocky <rocky@gnu.org>

 	* trepan/lib/breakpoint.py, trepan/lib/deparse.py,
-	trepan/lib/format.py, trepan/lib/printing.py, trepan/lib/stack.py:
+	trepan/lib/format.py, trepan/lib/printing.py, trepan/lib/stack.py:
 	Miscellaneous linting

 2025-08-10  rocky <rocky@gnu.org>
@@ -381,7 +381,7 @@

 2024-12-24  rocky <rocky@gnu.org>

-	* admin-tools/checkout_common.sh, admin-tools/setup-master.sh:
+	* admin-tools/checkout_common.sh, admin-tools/setup-master.sh:
 	Administrivia

 2024-12-24  rocky <rocky@gnu.org>
@@ -424,7 +424,7 @@

 2024-12-12  rocky <rocky@gnu.org>

-	* docs/commands/running/jump.rst, trepan/processor/command/jump.py:
+	* docs/commands/running/jump.rst, trepan/processor/command/jump.py:
 	Go over jump command docs

 2024-12-10  rocky <rocky@gnu.org>
@@ -434,12 +434,12 @@

 2024-12-10  rocky <rocky@gnu.org>

-	* admin-tools/setup-python-3.3.sh, admin-tools/setup-python-3.6.sh:
+	* admin-tools/setup-python-3.3.sh, admin-tools/setup-python-3.6.sh:
 	Administrivia

 2024-12-10  rocky <rocky@gnu.org>

-	* docs/conf.py, trepan/lib/core.py, trepan/processor/location.py:
+	* docs/conf.py, trepan/lib/core.py, trepan/processor/location.py:
 	Some small lint

 2024-12-09  rocky <rocky@gnu.org>
@@ -463,7 +463,7 @@
 	* docs/commands/data/eval.rst, docs/commands/running/jump.rst,
 	docs/commands/running/skip.rst, docs/commands/support/python.rst,
 	docs/manpages/trepan3kc.rst, trepan/processor/command/eval.py,
-	trepan/processor/command/jump.py, trepan/processor/command/skip.py:
+	trepan/processor/command/jump.py, trepan/processor/command/skip.py:
 	Go over jump, skip and eval, and python doc

 2024-12-09  rocky <rocky@gnu.org>
@@ -479,7 +479,7 @@
 	test/unit/processor/command/test_cmd_break.py,
 	trepan/lib/breakpoint.py, trepan/misc.py,
 	trepan/processor/cmdbreak.py, trepan/processor/cmdlist.py,
-	trepan/processor/command/list.py, trepan/processor/location.py:
+	trepan/processor/command/list.py, trepan/processor/location.py:
 	Revise breakpoint handling (#73) * Track down code in breakpoint command WARNING -- needs a new
 	filecache API for this.  * Fix up list command range parsing * Use github for pyficache in CI

@@ -558,7 +558,7 @@
 	docs/commands/running/continue.rst, docs/commands/running/exit.rst,
 	docs/commands/running/next.rst, docs/commands/running/step.rst,
 	docs/commands/running/stepi.rst, docs/commands/set/highlight.rst,
-	trepan/processor/command/next.py, trepan/processor/command/stepi.py:
+	trepan/processor/command/next.py, trepan/processor/command/stepi.py:
 	Go over docs

 2024-11-23  rocky <rocky@gnu.org>
@@ -636,7 +636,7 @@

 2024-11-14  rocky <rocky@gnu.org>

-	* admin-tools/checkout_common.sh, admin-tools/setup-python-3.3.sh:
+	* admin-tools/checkout_common.sh, admin-tools/setup-python-3.3.sh:
 	Administrivia

 2024-11-13  rocky <rocky@gnu.org>
@@ -657,7 +657,7 @@
 	admin-tools/setup-python-3.0.sh, docs/commands/set/asmfmt.rst,
 	docs/commands/set/events.rst, docs/commands/show/asmfmt.rst,
 	docs/commands/stack/frame.rst,
-	trepan/processor/command/info_subcmd/frame.py, trepan/version.py:
+	trepan/processor/command/info_subcmd/frame.py, trepan/version.py:
 	Get ready for release 1.3.0

 2024-11-12  rocky <rocky@gnu.org>
@@ -714,7 +714,7 @@
 2024-11-11  R. Bernstein <rocky@users.noreply.github.com>

 	* docs/entry-exit.rst, pyproject.toml, trepan/__main__.py,
-	trepan/api.py, trepan/client.py, trepan/processor/cmdproc.py:
+	trepan/api.py, trepan/client.py, trepan/processor/cmdproc.py:
 	Supoprt 3.7+ breakpoint() call (#63)

 2024-11-11  rocky <rocky@gnu.org>
@@ -748,7 +748,7 @@
 	trepan/processor/command/info_subcmd/locals.py,
 	trepan/processor/command/info_subcmd/macro.py,
 	trepan/processor/command/info_subcmd/signals.py,
-	trepan/processor/command/unalias.py, trepan/processor/complete.py:
+	trepan/processor/command/unalias.py, trepan/processor/complete.py:
 	Note "complete" is GNU-Readline complete

 2024-11-10  R. Bernstein <rocky@users.noreply.github.com>
@@ -837,7 +837,7 @@

 2024-10-14  rocky <rocky@gnu.org>

-	* admin-tools/setup-python-3.3.sh, admin-tools/setup-python-3.6.sh:
+	* admin-tools/setup-python-3.3.sh, admin-tools/setup-python-3.6.sh:
 	Admnisitrivia

 2024-10-09  rocky <rocky@gnu.org>
@@ -866,7 +866,7 @@
 	make-dist-3.3-3.5.sh}, admin-tools/merge-for-3.0.sh,
 	admin-tools/{merge-for-3.2.sh => merge-for-3.3.sh},
 	admin-tools/{pyenv-3.2-3.5-versions => pyenv-3.3-3.5-versions},
-	admin-tools/{setup-python-3.2.sh => setup-python-3.3.sh}:
+	admin-tools/{setup-python-3.2.sh => setup-python-3.3.sh}:
 	Administrivia

 2024-10-09  rocky <rocky@gnu.org>
@@ -905,7 +905,7 @@

 2024-09-29  rocky <rocky@gnu.org>

-	* trepan/lib/core.py, trepan/processor/command/info_subcmd/frame.py:
+	* trepan/lib/core.py, trepan/processor/command/info_subcmd/frame.py:
 	Small changes...  to "info frame" and trace_hook debugging

 2024-09-29  rocky <rocky@gnu.org>
@@ -943,9 +943,9 @@

 	* trepan/processor/command/load.py: expand load to handle top-level
 	modules...  Previously we only allowed a single module with a single command.
-	Now we can put serveral commands/modules under a single
+	Now we can put several commands/modules under a single
 	package/module.  For example we handle "load trepan3k_mathics3" in addition to the
-	previouly supported  "load mathics3k_mathics3.printelement"
+	previously supported  "load mathics3k_mathics3.printelement"

 2024-09-22  rocky <rocky@gnu.org>

@@ -1015,7 +1015,7 @@

 2024-09-21  rocky <rocky@gnu.org>

-	* admin-tools/pyenv-3.6-3.10-versions, test/functional/test_step.py:
+	* admin-tools/pyenv-3.6-3.10-versions, test/functional/test_step.py:
 	Changes for 3.6-3.10 branch

 2024-09-21  R. Bernstein <rocky@users.noreply.github.com>
@@ -1049,7 +1049,7 @@
 	* Makefile, test/integration/test-highlight.py,
 	test/integration/test-noscript.py,
 	test/integration/test_highlight.py,
-	test/integration/test_noscript.py, test/integration/test_step.py:
+	test/integration/test_noscript.py, test/integration/test_step.py:
 	Get integration tests working again

 2024-09-15  rocky <rocky@gnu.org>
@@ -1139,7 +1139,7 @@

 2024-08-28  rocky <rocky@gnu.org>

-	* admin-tools/setup-python-3.0.sh, admin-tools/setup-python-3.6.sh:
+	* admin-tools/setup-python-3.0.sh, admin-tools/setup-python-3.6.sh:
 	Administrivia

 2024-08-26  rocky <rocky@gnu.org>
@@ -1205,7 +1205,7 @@

 2024-08-25  rocky <rocky@gnu.org>

-	* trepan/processor/cmdproc.py, trepan/processor/command/frame.py:
+	* trepan/processor/cmdproc.py, trepan/processor/command/frame.py:
 	small tweaks...  Use thread.getName -> name since former is deprecated I messed up in
 	comment on what curindex represents Initialize position_str

@@ -1226,7 +1226,7 @@
 2024-08-24  rocky <rocky@gnu.org>

 	* trepan/lib/format.py, trepan/lib/stack.py,
-	trepan/processor/command/frame.py, trepan/processor/frame.py:
+	trepan/processor/command/frame.py, trepan/processor/frame.py:
 	Modernize a little ...  more f-strings, more import from's, add more annotations

 2024-08-24  rocky <rocky@gnu.org>
@@ -1332,7 +1332,7 @@

 2024-07-25  rocky <rocky@gnu.org>

-	* trepan/processor/command/p.py, trepan/processor/command/pp.py:
+	* trepan/processor/command/p.py, trepan/processor/command/pp.py:
 	Misc

 2024-07-25  rocky <rocky@gnu.org>
@@ -1352,12 +1352,12 @@

 	* admin-tools/make-dist-3.0-3.1.sh,
 	admin-tools/make-dist-3.2-3.5.sh, admin-tools/setup-master.sh,
-	admin-tools/setup-python-3.0.sh, admin-tools/setup-python-3.6.sh:
+	admin-tools/setup-python-3.0.sh, admin-tools/setup-python-3.6.sh:
 	Administrivia

 2024-07-22  rocky <rocky@gnu.org>

-	* admin-tools/setup-master.sh, admin-tools/setup-python-3.6.sh:
+	* admin-tools/setup-master.sh, admin-tools/setup-python-3.6.sh:
 	Administrivia

 2024-07-22  rocky <rocky@gnu.org>
@@ -1436,7 +1436,7 @@
 	* docs/commands/data/disassemble.rst, trepan/lib/bytecode.py,
 	trepan/lib/default.py, trepan/lib/format.py,
 	trepan/processor/command/disassemble.py: Fix some of the funky
-	terminal highight problems..  more remain. Doc asmft
+	terminal highlight problems..  more remain. Doc asmft

 2024-03-06  rocky <rocky@gnu.org>

@@ -1488,14 +1488,14 @@

 2024-02-11  rocky <rocky@gnu.org>

-	* admin-tools/{check-newer-versions.sh => check-newest-versions.sh}:
+	* admin-tools/{check-newer-versions.sh => check-newest-versions.sh}:
 	Rename check program to something better

 2024-02-11  rocky <rocky@gnu.org>

 	* test/functional/test_break.py, test/functional/test_finish.py,
 	test/functional/test_jump.py, test/functional/test_sig.py,
-	test/functional/test_skip.py: Remove exectuable mode bit
+	test/functional/test_skip.py: Remove executable mode bit

 2024-02-11  rocky <rocky@gnu.org>

@@ -1541,12 +1541,12 @@

 2024-02-05  rocky <rocky@gnu.org>

-	* __pkginfo__.py, test/unit/test_api.py: 3.10 compatability and
+	* __pkginfo__.py, test/unit/test_api.py: 3.10 compatibility and
 	uncompyle6 selection

 2024-02-05  rocky <rocky@gnu.org>

-	* test/functional/test-next.py, test/functional/test_next.py:
+	* test/functional/test-next.py, test/functional/test_next.py:
 	Convert test_next.py

 2024-02-04  rocky <rocky@gnu.org>
@@ -1637,7 +1637,7 @@
 	test/unit/inout/test_inout_input.py,
 	test/unit/processor/test_proc_subcmd.py, test/unit/test-fifo.py,
 	test/unit/test-inout-input.py, test/unit/test-subcmd.py,
-	trepan/processor/command/set_subcmd/style.py: Convert remainging
+	trepan/processor/command/set_subcmd/style.py: Convert remaining
 	unit tests ...  and reduce nose dependency.

 2024-01-28  rocky <rocky@gnu.org>
@@ -1703,7 +1703,7 @@
 	* test/unit/processor/command/test_cmd_alias.py,
 	test/unit/processor/command/test_cmd_info_files.py,
 	test/unit/processor/command/test_cmd_list.py,
-	test/unit/test-cmd-alias.py, trepan/processor/command/alias.py:
+	test/unit/test-cmd-alias.py, trepan/processor/command/alias.py:
 	Convert "alias" unit test to pytest

 2024-01-23  rocky <rocky@gnu.org>
@@ -1728,7 +1728,7 @@

 2024-01-16  rocky <rocky@gnu.org>

-	* test/unit/lib/test_lib_stack.py, test/unit/lib/test_lib_thread.py:
+	* test/unit/lib/test_lib_stack.py, test/unit/lib/test_lib_thread.py:
 	Lint more

 2024-01-16  rocky <rocky@gnu.org>
@@ -1812,7 +1812,7 @@
 	test/unit/test_cmd_break.py, test/unit/test_cmd_finish.py,
 	test/unit/test_cmd_run.py, test/unit/test_lib_bytecode.py,
 	test/unit/test_quit.py, trepan/interfaces/user.py,
-	trepan/processor/command/quit.py, trepan/processor/command/run.py:
+	trepan/processor/command/quit.py, trepan/processor/command/run.py:
 	More unit tests converted to pytest

 2024-01-15  rocky <rocky@gnu.org>
@@ -1835,7 +1835,7 @@
 	trepan/lib/default.py, trepan/lib/deparse.py,
 	trepan/lib/disassemble.py, trepan/lib/file.py,
 	trepan/lib/format.py, trepan/lib/pp.py, trepan/lib/printing.py,
-	trepan/lib/sighandler.py, trepan/lib/stack.py, trepan/lib/thred.py:
+	trepan/lib/sighandler.py, trepan/lib/stack.py, trepan/lib/thred.py:
 	Another pass over trepan/lib

 2024-01-14  rocky <rocky@gnu.org>
@@ -1869,7 +1869,7 @@

 2024-01-14  rocky <rocky@gnu.org>

-	* test/unit/test-completion.py, test/unit/test_completion.py:
+	* test/unit/test-completion.py, test/unit/test_completion.py:
 	Another test converted

 2024-01-14  rocky <rocky@gnu.org>
@@ -1926,12 +1926,12 @@

 2023-12-17  rocky <rocky@gnu.org>

-	* .gitignore, test/unit/test-lib-sig.py, trepan/lib/sighandler.py:
+	* .gitignore, test/unit/test-lib-sig.py, trepan/lib/sighandler.py:
 	Small lint corrections

 2023-09-04  rocky <rocky@gnu.org>

-	* admin-tools/pyenv-newer-versions, admin-tools/setup-master.sh:
+	* admin-tools/pyenv-newer-versions, admin-tools/setup-master.sh:
 	Bump testing versions

 2023-09-04  rocky <rocky@gnu.org>
@@ -1945,13 +1945,13 @@

 2023-07-01  rocky <rocky@gnu.org>

-	* admin-tools/setup-master.sh, admin-tools/setup-python-3.2.sh:
+	* admin-tools/setup-master.sh, admin-tools/setup-python-3.2.sh:
 	Checkout of 3.2 branch was missing pyficache and xdis

 2023-06-10  rocky <rocky@gnu.org>

 	* .pre-commit-config.yaml, admin-tools/pyenv-newer-versions,
-	admin-tools/setup-master.sh, trepan/processor/command/deparse.py:
+	admin-tools/setup-master.sh, trepan/processor/command/deparse.py:
 	Bump testing package versions

 2023-06-10  rocky <rocky@gnu.org>
@@ -2255,7 +2255,7 @@

 2021-11-25  rocky <rocky@gnu.org>

-	* admin-tools/pyenv-newer-versions, trepan/processor/cmdfns.py:
+	* admin-tools/pyenv-newer-versions, trepan/processor/cmdfns.py:
 	Remove PYTHON_VERSION

 2021-11-19  rocky <rocky@gnu.org>
@@ -2273,7 +2273,7 @@

 2021-10-30  rocky <rocky@gnu.org>

-	* admin-tools/setup-master.sh, admin-tools/setup-python-3.2.sh:
+	* admin-tools/setup-master.sh, admin-tools/setup-python-3.2.sh:
 	Admnistrivia

 2021-10-30  R. Bernstein <rocky@users.noreply.github.com>
@@ -2300,7 +2300,7 @@

 	* __pkginfo__.py, docs/conf.py, setup.py,
 	trepan/lib/disassemble.py, trepan/lib/stack.py,
-	trepan/processor/command/deparse.py, trepan/processor/location.py:
+	trepan/processor/command/deparse.py, trepan/processor/location.py:
 	More newer xdis compatibility

 2021-09-30  rocky <rocky@gnu.org>
@@ -2463,7 +2463,7 @@

 2021-03-15  rocky <rocky@gnu.org>

-	* admin-tools/how-to-make-a-release.md, trepan/processor/cmdproc.py:
+	* admin-tools/how-to-make-a-release.md, trepan/processor/cmdproc.py:
 	Fix bug encountered in `set patmap` The code is I think a holdover from some earlier code that no longer
 	exists.

@@ -2473,7 +2473,7 @@
 	admin-tools/how-to-make-a-release.md, admin-tools/pyenv-versions,
 	setup.py, tox.ini, trepan/__init__.py, trepan/__main__.py,
 	trepan/client.py, trepan/processor/command/__init__.py,
-	trepan/processor/command/set_subcmd/tempdir.py, trepan/version.py:
+	trepan/processor/command/set_subcmd/tempdir.py, trepan/version.py:
 	Get ready for release 1.2.3

 2021-01-24  rocky <rocky@gnu.org>
@@ -2672,7 +2672,7 @@

 2020-06-16  rocky <rocky@gnu.org>

-	* trepan/processor/cmdbreak.py, trepan/processor/location.py:
+	* trepan/processor/cmdbreak.py, trepan/processor/location.py:
 	breakpoint-handling fixes...  have to back off adding function into breakpoint structure.
 	Location is returning an "int" sometimes. Need to investigate, but
 	for now...
@@ -2780,14 +2780,14 @@
 	test/unit/test-cmdbreak.py, trepan/lib/breakpoint.py,
 	trepan/processor/cmdbreak.py, trepan/processor/command/break.py,
 	trepan/processor/command/info_subcmd/line.py,
-	trepan/processor/location.py, trepan/processor/parse/semantics.py:
+	trepan/processor/location.py, trepan/processor/parse/semantics.py:
 	Start adding/reporting code offsets in debugger

 2020-06-11  rocky <rocky@gnu.org>

 	* .circleci/config.yml, .travis.yml, __pkginfo__.py,
-	requirements.txt, trepan/processor/command/info_subcmd/line.py:
-	Expanded xdis line/offset in "info line"...  using information from xdis. We can now describe * What other offsets are stoppable for that line * what function the line is in * The starting offset and how many intructions are in the line
+	requirements.txt, trepan/processor/command/info_subcmd/line.py:
+	Expanded xdis line/offset in "info line"...  using information from xdis. We can now describe * What other offsets are stoppable for that line * what function the line is in * The starting offset and how many instructions are in the line

 2020-06-10  rocky <rocky@gnu.org>

@@ -2852,7 +2852,7 @@
 	* trepan/api.py, trepan/processor/command/base_cmd.py,
 	trepan/processor/command/continue.py,
 	trepan/processor/command/deparse.py,
-	trepan/processor/command/next.py, trepan/processor/command/show.py:
+	trepan/processor/command/next.py, trepan/processor/command/show.py:
 	Blacken and update trepan.api DRY, DebuggerCommand class vars initialization

 2020-06-04  rocky <rocky@gnu.org>
@@ -3012,7 +3012,7 @@
 2020-05-14  rocky <rocky@gnu.org>

 	* trepan/inout/base.py, trepan/inout/output.py,
-	trepan/lib/default.py, trepan/lib/pp.py, trepan/lib/printing.py:
+	trepan/lib/default.py, trepan/lib/pp.py, trepan/lib/printing.py:
 	Blacken and lint some files

 2020-05-14  rocky <rocky@gnu.org>
@@ -3029,7 +3029,7 @@

 2020-05-13  rocky <rocky@gnu.org>

-	* README.rst, docs/commands/show/autopc.rst, docs/features.rst:
+	* README.rst, docs/commands/show/autopc.rst, docs/features.rst:
 	Update Docs yet again.

 2020-05-13  rocky <rocky@gnu.org>
@@ -3086,7 +3086,7 @@

 	* test/unit/test-lib-format.py, trepan/lib/deparse.py,
 	trepan/processor/command/continue.py,
-	trepan/processor/command/quit.py, trepan/processor/command/step.py:
+	trepan/processor/command/quit.py, trepan/processor/command/step.py:
 	Misc modernization & skip a test...  Something in reflowing adds spaces. Don't need/want to deal with
 	that now.

@@ -3185,7 +3185,7 @@
 2020-04-24  rocky <rocky@gnu.org>

 	* .circleci/config.yml, .travis.yml, __pkginfo__.py: Bump xdis
-	version now that a new verison is out
+	version now that a new version is out

 2020-04-22  R. Bernstein <rocky@users.noreply.github.com>

@@ -3222,7 +3222,7 @@

 2020-03-16  rocky <rocky@gnu.org>

-	* __pkginfo__.py, admin-tools/how-to-make-a-release.md, setup.py:
+	* __pkginfo__.py, admin-tools/how-to-make-a-release.md, setup.py:
 	Administrivia release-type fixes

 2020-03-16  rocky <rocky@gnu.org>
@@ -3253,7 +3253,7 @@

 2019-11-16  rocky <rocky@gnu.org>

-	* admin-tools/pyenv-newer-versions, admin-tools/setup-master.sh:
+	* admin-tools/pyenv-newer-versions, admin-tools/setup-master.sh:
 	Administriva - bump testing versions

 2019-11-11  rocky <rocky@gnu.org>
@@ -3304,7 +3304,7 @@

 2019-09-30  rocky <rocky@gnu.org>

-	* docs/entry-exit.rst, trepan/api.py, trepan/processor/cmdproc.py:
+	* docs/entry-exit.rst, trepan/api.py, trepan/processor/cmdproc.py:
 	Misc IPython-related and doc fixes

 2019-09-30  rocky <rocky@gnu.org>
@@ -3658,7 +3658,7 @@

 2018-03-21  rocky <rocky@gnu.org>

-	* trepan/lib/disassemble.py, trepan/processor/command/deparse.py:
+	* trepan/lib/disassemble.py, trepan/processor/command/deparse.py:
 	Python sourcecode lines go into 1,000s...  So save 4 places for these.

 2018-03-13  rocky <rocky@gnu.org>
@@ -3705,7 +3705,7 @@

 2018-02-16  rocky <rocky@gnu.org>

-	* __pkginfo__.py, setup.py, trepan/processor/command/disassemble.py:
+	* __pkginfo__.py, setup.py, trepan/processor/command/disassemble.py:
 	disassemble improvements

 2018-02-11  rocky <rocky@gnu.org>
@@ -3796,7 +3796,7 @@

 	* trepan/lib/deparse.py, trepan/lib/disassemble.py,
 	trepan/processor/cmdfns.py, trepan/processor/cmdproc.py: Start
-	deparse line remaping
+	deparse line remapping

 2018-01-22  rocky <rocky@gnu.org>

@@ -3813,7 +3813,7 @@

 2018-01-21  rocky <rocky@gnu.org>

-	* admin-tools/how-to-make-a-release.md, admin-tools/make-dist.sh:
+	* admin-tools/how-to-make-a-release.md, admin-tools/make-dist.sh:
 	Admin tweak

 2018-01-21  rocky <rocky@gnu.org>
@@ -3844,7 +3844,7 @@

 2018-01-19  rocky <rocky@gnu.org>

-	* trepan/processor/cmdbreak.py, trepan/processor/command/break.py:
+	* trepan/processor/cmdbreak.py, trepan/processor/command/break.py:
 	Add force option on break

 2018-01-18  rocky <rocky@gnu.org>
@@ -4072,7 +4072,7 @@
 2017-08-10  rocky <rocky@gnu.org>

 	* celery/README.md, celery/{trepan.py => ctrepan.py},
-	trepan/interfaces/client.py, trepan/processor/command/python.py:
+	trepan/interfaces/client.py, trepan/processor/command/python.py:
 	Bang on celery remote debugging and...  Improve exception handling when going into python: don't go into
 	post-mortem debugging.

@@ -4103,7 +4103,7 @@
 2017-08-04  rocky <rocky@gnu.org>

 	* celery/trepan.py, trepan/inout/tcpserver.py,
-	trepan/interfaces/server.py, trepan/processor/command/python.py:
+	trepan/interfaces/server.py, trepan/processor/command/python.py:
 	Bang on remote execution more

 2017-08-03  rocky <rocky@gnu.org>
@@ -4268,7 +4268,7 @@

 	* .gitignore, MANIFEST.in, setup.py,
 	test/data/highlight-pyp35.right, test/data/macro-pypy35.right,
-	test/integration/.gitignore, trepan/__init__.py, trepan/api.py:
+	test/integration/.gitignore, trepan/__init__.py, trepan/api.py:
 	Revise packaging and Administrivia

 2017-05-10  rocky <rocky@gnu.org>
@@ -4389,7 +4389,7 @@

 2016-11-12  rocky <rocky@gnu.org>

-	* trepan/processor/cmdproc.py, trepan/processor/command/list.py:
+	* trepan/processor/cmdproc.py, trepan/processor/command/list.py:
 	Workaround check_path_with_frame problems

 2016-11-12  rocky <rocky@gnu.org>
@@ -4398,7 +4398,7 @@

 2016-11-12  rocky <rocky@gnu.org>

-	* trepan/processor/cmdproc.py, trepan/processor/command/list.py:
+	* trepan/processor/cmdproc.py, trepan/processor/command/list.py:
 	Ease into source file checking against code

 2016-11-12  rocky <rocky@gnu.org>
@@ -4440,7 +4440,7 @@

 2016-10-06  rocky <rocky@gnu.org>

-	* .travis.yml, setup.py, trepan/{VERSION.py => version.py}:
+	* .travis.yml, setup.py, trepan/{VERSION.py => version.py}:
 	Administrivia and Travis * setup.py, version.py: Don't use exec.  * .travis.yml: Add "python ./setup.py develop"

 2016-10-06  rocky <rocky@gnu.org>
@@ -4464,7 +4464,7 @@

 	* trepan/processor/command/display.py,
 	trepan/processor/command/set_subcmd/highlight.py: display
-	expresssion and highlight changes...  * show display expression after setting it * clear source-code cache after setting highlight
+	expression and highlight changes...  * show display expression after setting it * clear source-code cache after setting highlight

 2016-07-26  rocky <rocky@gnu.org>

@@ -4516,7 +4516,7 @@
 2016-05-01  rocky <rocky@gnu.org>

 	* Makefile, trepan/inout/tcpclient.py, trepan/inout/tcpfns.py,
-	trepan/inout/tcpserver.py, trepan/processor/command/disassemble.py:
+	trepan/inout/tcpserver.py, trepan/processor/command/disassemble.py:
 	flake8 crap

 2016-05-01  rocky <rocky@gnu.org>
@@ -4555,7 +4555,7 @@

 2016-01-13  rocky <rocky@gnu.org>

-	* tox.ini, trepan/bwprocessor/main.py, trepan/lib/sighandler.py:
+	* tox.ini, trepan/bwprocessor/main.py, trepan/lib/sighandler.py:
 	Squelch flake8 complaints

 2016-01-13  rocky <rocky@gnu.org>
@@ -4677,7 +4677,7 @@

 2015-12-20  rocky <rocky@gnu.org>

-	* test/example/gcd-dbgcall.py, trepan/processor/command/deparse.py:
+	* test/example/gcd-dbgcall.py, trepan/processor/command/deparse.py:
 	Add deparse command

 2015-12-18  rocky <rocky@gnu.org>
@@ -4718,7 +4718,7 @@

 2015-12-06  rocky <rocky@gnu.org>

-	* README.rst, trepan/lib/stack.py, trepan/processor/cmdproc.py:
+	* README.rst, trepan/lib/stack.py, trepan/processor/cmdproc.py:
 	Small thiungs. Sync code with python2

 2015-12-02  rocky <rocky@gnu.org>
@@ -4794,7 +4794,7 @@
 2015-11-28  rocky <rocky@gnu.org>

 	* bin/trepan3k, trepan/lib/format.py, trepan/lib/printing.py,
-	trepan/options.py: options.py: Move configiration loation to
+	trepan/options.py: options.py: Move configuration loation to
 	~/.config. Add hook for python code format.py: and dark/light
 	reversed. Customize colorstyle more printing.py: address Python2/3
 	incompatibility bin/trepan3k: invocation convenience script
@@ -4862,7 +4862,7 @@

 	* NEWS, README.rst, trepan/VERSION.py,
 	trepan/processor/command/exit.py, trepan/processor/command/kill.py,
-	trepan/processor/command/mock.py, trepan/processor/command/quit.py:
+	trepan/processor/command/mock.py, trepan/processor/command/quit.py:
 	Get ready for release 0.5.3

 2015-09-23  rocky <rocky@gnu.org>
@@ -4986,7 +4986,7 @@
 	trepan/processor/command/show_subcmd/aliases.py,
 	trepan/processor/command/show_subcmd/autoeval.py,
 	trepan/processor/command/skip.py, trepan/processor/command/step.py,
-	trepan/processor/command/up.py, trepan/processor/command/whatis.py:
+	trepan/processor/command/up.py, trepan/processor/command/whatis.py:
 	Go over command docstrings. Sync with trepan2

 2015-05-29  rocky <rocky@gnu.org>
@@ -5000,7 +5000,7 @@
 2015-05-29  rocky <rocky@gnu.org>

 	* README.rst, trepan/lib/eval.py, trepan/post_mortem.py,
-	trepan/processor/command/bpy.py, trepan/processor/command/python.py:
+	trepan/processor/command/bpy.py, trepan/processor/command/python.py:
 	Add bpython shell. eval? of for VAR in EXPR: is EXPR

 2015-05-27  rocky <rocky@gnu.org>
@@ -5039,7 +5039,7 @@

 	* NEWS, trepan/lib/file.py, trepan/processor/command/break.py,
 	trepan/processor/command/info_subcmd/files.py,
-	trepan/processor/command/tbreak.py, trepan/processor/complete.py:
+	trepan/processor/command/tbreak.py, trepan/processor/complete.py:
 	Add line completion on tbreak and break

 2015-05-19  rocky <rocky@gnu.org>
@@ -5055,7 +5055,7 @@
 	trepan/processor/command/info_subcmd/program.py,
 	trepan/processor/command/info_subcmd/signals.py,
 	trepan/processor/command/info_subcmd/threads.py: Add "info *" and
-	"info arg1, arg2". Update docs Add "info frame *number* set/sheck
+	"info arg1, arg2". Update docs Add "info frame *number* set/check
 	max args in subcommands

 2015-05-17  rocky <rocky@gnu.org>
@@ -5109,7 +5109,7 @@
 	trepan/processor/command/display.py,
 	trepan/processor/command/pp.py, trepan/processor/command/pr.py,
 	trepan/processor/command/pydocx.py,
-	trepan/processor/command/whatis.py, trepan/processor/complete.py:
+	trepan/processor/command/whatis.py, trepan/processor/complete.py:
 	Add identifier completions on pp, pr, pydocx, and debug Some
 	commands include builtins and some do not

@@ -5121,7 +5121,7 @@
 	trepan/processor/command/disable.py,
 	trepan/processor/command/enable.py,
 	trepan/processor/command/info_subcmd/break.py,
-	trepan/processor/complete.py: Add compeltion for breakpoint numbers
+	trepan/processor/complete.py: Add completion for breakpoint numbers

 2015-05-16  rocky <rocky@gnu.org>

@@ -5179,7 +5179,7 @@

 2015-05-15  rocky <rocky@gnu.org>

-	* __pkginfo__.py, trepan/debugger.py, trepan/processor/cmdproc.py:
+	* __pkginfo__.py, trepan/debugger.py, trepan/processor/cmdproc.py:
 	Correct <string> remapping. Need new version of pifycache though

 2015-05-14  rocky <rocky@gnu.org>
@@ -5225,7 +5225,7 @@

 2015-05-01  rocky <rocky@gnu.org>

-	* trepan/processor/cmdproc.py, trepan/processor/command/ipython.py:
+	* trepan/processor/cmdproc.py, trepan/processor/command/ipython.py:
 	Lint errors

 2015-05-01  rocky <rocky@gnu.org>
@@ -5290,11 +5290,11 @@

 2015-04-06  rocky <rocky@gnu.org>

-	* README.rst: README.rst link correcton
+	* README.rst: README.rst link correction

 2015-04-06  rocky <rocky@gnu.org>

-	* README.rst, trepan/VERSION.py: README.rst link correcton
+	* README.rst, trepan/VERSION.py: README.rst link correction

 2015-04-06  rocky <rocky@gnu.org>

@@ -5328,7 +5328,7 @@

 2015-04-06  rocky <rocky@gnu.org>

-	* test/integration/test-general.py, test/integration/test-macro.py:
+	* test/integration/test-general.py, test/integration/test-macro.py:
 	Travis 8th try

 2015-04-06  rocky <rocky@gnu.org>
@@ -5349,7 +5349,7 @@

 2015-04-06  rocky <rocky@gnu.org>

-	* .travis.yml, requirements-dev.txt, test/functional/test-skip.py:
+	* .travis.yml, requirements-dev.txt, test/functional/test-skip.py:
 	Travis 3rd try

 2015-04-06  rocky <rocky@gnu.org>
@@ -5567,8 +5567,8 @@

 	* test/data/macro.right, test/data/step.right, test/example/gcd.py,
 	test/functional/fn_helper.py, trepan/options.py:  Remove trepan2-client from console_scripts (Georg Brandl)
-	Integeration test output changes due to running into defaultoutput
-	encoding. FIXME down the line. Setup file name creation wrong becase
+	Integration test output changes due to running into defaultoutput
+	encoding. FIXME down the line. Setup file name creation wrong because
 	debugger name is "trepan" not trepan3k.

 2015-04-04  rocky <rocky@gnu.org>
@@ -5613,7 +5613,7 @@

 	* test/unit/test-cmd-alias.py, test/unit/test-pdef.py,
 	trepan/debugger.py, trepan/processor/command/frame.py,
-	trepan/processor/command/pdef.py: Refert sync change with pdef
+	trepan/processor/command/pdef.py: Revert sync change with pdef

 2015-01-24  Rocky Bernstein <rocky@gnu.org>

@@ -5740,7 +5740,7 @@
 	trepan/processor/command/undisplay.py,
 	trepan/processor/complete.py, trepan/processor/frame.py,
 	trepan/processor/subcmd.py, trepan/processor/trace.py,
-	trepan/vprocessor.py: First attempt to syncronize with trepan2 from
+	trepan/vprocessor.py: First attempt to synchronize with trepan2 from
 	6 months ago.

 2014-07-04  Rocky Bernstein <rocky@gnu.org>
@@ -5779,7 +5779,7 @@
 2014-02-03  Rocky Bernstein <rocky@gnu.org>

 	* test/functional/fn_helper.py, trepan/cli.py, trepan/client.py,
-	trepan/options.py, trepan/processor/command/info_subcmd/files.py:
+	trepan/options.py, trepan/processor/command/info_subcmd/files.py:
 	Start work on --host and --port options for remote execution.

 2013-11-30  Rocky Bernstein <rocky@gnu.org>
@@ -5986,7 +5986,7 @@

 	* trepan/io/fifoclient.py, trepan/io/fifoserver.py,
 	trepan/io/input.py, trepan/io/output.py, trepan/io/scriptin.py,
-	trepan/io/tcpclient.py, trepan/io/tcpserver.py, trepan/lib/eval.py:
+	trepan/io/tcpclient.py, trepan/io/tcpserver.py, trepan/lib/eval.py:
 	Fix macro processing error messages better. pyflakes io directory.

 2013-03-23  Rocky Bernstein <rocky@gnu.org>
@@ -6094,7 +6094,7 @@
 2013-03-22  Rocky Bernstein <rocky@gnu.org>

 	* test/unit/test-disassemble.py, trepan/io/stringarray.py,
-	trepan/lib/disassemble.py, trepan/processor/command/disassemble.py:
+	trepan/lib/disassemble.py, trepan/processor/command/disassemble.py:
 	Fix bugs in disassemble command and add test for it.

 2013-03-22  Rocky Bernstein <rocky@gnu.org>
@@ -6252,7 +6252,7 @@
 	trepan/processor/command/show_subcmd/basename.py,
 	trepan/processor/command/show_subcmd/confirm.py,
 	trepan/processor/command/tbreak.py,
-	trepan/processor/command/undisplay.py: First installable and workin
+	trepan/processor/command/undisplay.py: First installable and working
 	version. Finish show subcommands. Install trepan3. Add
 	"until_condition" to step until we hit __main__ to skip over
 	encoding stuff.
@@ -6302,7 +6302,7 @@
 2013-03-13  Rocky Bernstein <rocky@gnu.org>

 	* trepan/processor/command/display.py,
-	trepan/processor/command/eval.py: Knock off another two debuger
+	trepan/processor/command/eval.py: Knock off another two debugger
 	commands

 2013-03-13  Rocky Bernstein <rocky@gnu.org>
@@ -6322,7 +6322,7 @@
 2013-03-12  Rocky Bernstein <rocky@gnu.org>

 	* trepan/processor/cmdproc.py, trepan/processor/command/down.py,
-	trepan/processor/command/frame.py, trepan/processor/command/up.py:
+	trepan/processor/command/frame.py, trepan/processor/command/up.py:
 	Add "up", "down" and "frame" commands

 2013-03-12  Rocky Bernstein <rocky@gnu.org>
@@ -6343,12 +6343,12 @@
 	trepan/processor/cmdproc.py, trepan/processor/command/backtrace.py,
 	trepan/processor/command/continue.py,
 	trepan/processor/command/help.py, trepan/processor/command/list.py,
-	trepan/processor/command/next.py, trepan/processor/command/step.py:
+	trepan/processor/command/next.py, trepan/processor/command/step.py:
 	Debugger starts working! Start to fill in the vast number commands.

 2013-03-11  rocky <rocky@gnu.org>

-	* trepan/api.py, trepan/post_mortem.py, trepan/processor/cmdproc.py:
+	* trepan/api.py, trepan/post_mortem.py, trepan/processor/cmdproc.py:
 	Add api and post-mortem

 2013-03-11  Rocky Bernstein <rocky@gnu.org>
@@ -6402,7 +6402,7 @@

 	* test/unit/cmdhelper.py, trepan/interface.py,
 	trepan/interfaces/__init__.py, trepan/interfaces/user.py: Start
-	converting interface code, and user inteface in particular.
+	converting interface code, and user interface in particular.

 2013-03-07  rocky <rocky@gnu.org>

@@ -6435,14 +6435,14 @@
 	* .gitignore, Makefile, test/unit/test-io-input.py,
 	trepan/__init__.py, trepan/io/__init__.py, trepan/io/{dbg_input.py
 	=> input.py}, trepan/io/{dbg_output.py => output.py},
-	trepan/io/scriptin.py, trepan/lib/__init__.py, trepan/misc.py:
+	trepan/io/scriptin.py, trepan/lib/__init__.py, trepan/misc.py:
 	dbg_{input,output} -> {input,output}. misc.py, __init__.py: start to
-	simplfy and DRY code
+	simpilfy and DRY code

 2013-02-24  rocky <rocky@gnu.org>

 	* .gitignore, Makefile, NEWS, README.txt, __init__.py,
-	__pkginfo__.py, make-check-filter.py, setup.py, trepan/VERSION.py:
+	__pkginfo__.py, make-check-filter.py, setup.py, trepan/VERSION.py:
 	Package Administrivia

 2013-02-24  rocky <rocky@gnu.org>
@@ -6455,4 +6455,3 @@
 2013-02-24  rocky <rocky@gnu.org>

 	* Convert more stuff
-
